Source/santabundleservice/SNTBundleService.mm

```diff
@@ -139,11 +164,12 @@ - (NSDictionary *)findRelatedBinaries:(SNTStoredEvent *)event progress:(NSProgre
   //
   // Xcode.app has roughly 500k files, 8bytes per pointer is ~4MB for this array. This size to space
   // ratio seems appropriate as Xcode.app is in the upper bounds of bundle size.
-  __block void **fis = calloc(subpaths.count, sizeof(void *));
+  __block void **fis = static_cast<void **>(calloc(subpaths.count, sizeof(void *)));
```

Comment:
Not a blocker, but might be nice to consider an `std::vector` since this is an ObjC++ file now - then we wouldn't have to worry about freeing it manually.